### DISCUSSION TITLE
How can i disable dragging/moving  AR anchor.

### DISCUSSION THREAD

**Discussion by Hanif-MI** [sceneview/sceneview#518] [Q&A]
@ThomasGorisse How can I disable dragging/moving  AR anchor?

**Comment by ThomasGorisse** [comment]
In v4.0.0-rc.1, anchors are immovable by default. Once you create an `AnchorNode`, it stays fixed at its world position — the user can't drag it unless you explicitly implement drag handling.

```kotlin
ARSceneView {
    AnchorNode(anchor = anchor) {
        ModelNode(modelInstance = model) // tappable but not draggable by default
    }
}
```

If you had drag enabled from a previous version's config, just remove any custom gesture handling that repositions the anchor.
